### DISCUSSION TITLE
Shared Albums are not backing up

### DISCUSSION THREAD

**Discussion by vdevan** [gilesknap/gphotos-sync#403] [Q&A] [answered]
I have few shared albums from families. When I download them using gphotos-sync, all the photos are getting downloaded from the shared albums and stored in photos-path. But the shared album name and structure itself is not getting downloaded. Is this something to do with API or something to be expected?
Any help would be appreciated. Thanks

**Comment by gilesknap** [comment]
@vdevan I'm not sure if you are describing:
a) albums that exist in another family member's library and you have switched on library sharing with that family member
b) a standard shared album that exists in your library with photos contributed from other family members.

I have (b) in my library and it works as expected. I do also have library sharing on but I have not checked if albums in other family members libraries are copied (because they don't have albums at present).

I can check (a) if that is your scenario.

**Comment by vdevan** [comment]
Thanks for your response Giles. My scenario is a. Albums that exist with
another family members. They share their albums with me. I could download
all the photos with —hardlink switch. But Album names are not getting
downloaded. I can send the error message if that helps. It is some 256 hex
values looks like crypt key.

On Thu, 5 Jan 2023 at 4:32 pm, Giles Knap ***@***.***> wrote:

> @vdevan <https://github.com/vdevan> I'm not sure if you are describing:
> a) albums that exist in another family member's library and you have
> switched on library sharing with that family member
> b) a standard shared album that exists in your library with photos
> contributed from other family members.
>
> I have (b) in my library and it works as expected. I do also have library
> sharing on but I have not checked if albums in other family members
> libraries are copied (because they don't have albums at present).
>
> I can check (a) if that is your scenario.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gilesknap/gphotos-sync/discussions/403#discussioncomment-4598093>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEVDSMPFM4DE3NIF3ZSZPYLWQZMIBANCNFSM6AAAAAATRLENPQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Always lovingly
Vasu Devan V.
God give me strength to love everyone.
https://blogs.kamban.com.au

**Comment by gilesknap** [comment]
Yes please send the error message you are seeing.

Your scenario should work - I'm doing that on my own photos library.

**Comment by vdevan** [comment, marked as answer]
G'day Giles:
Sorry for raising the issues. I just checked my cron job today and I was
able to see all the albums today. Earlier I used flush-index and rescan
switch from console and albums never got downloaded. Not sure what fixed
it. But it did work today. Maybe some indexing at Google photos since I had
to delete all the files after using 'Google Takeout'.


என்றும் அன்புடன்
வாசு
அன்பைத் தவிர வேறு ஒன்றும் வேண்டேன் பராபரமே!


Always lovingly
Vasu Devan V.
God give me strength to love everyone.
http://www.kamban.com.au
